documentation/release_notes.rst

```diff
@@ -8,6 +8,33 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6.1
+===============
+
+Fixed Issues
+------------
+- Fixed compilation errors happening when using C++20.
+- Fixed ``CL_OUT_OF_RESOURCES`` issue which used to occur during execution of Radix sort algorithm on CPU devices.
```

Comment:
May be "Fixed ``CL_OUT_OF_RESOURCES`` issue for Radix sort algorithm executed on DPC++ CPU devices"?

Reply:
Applied Dmitriy's suggestion